moped-database/metadata/tables.yaml

```diff
@@ -5019,6 +5019,7 @@
           - picture
           - roles
           - title
+          - user_id
```

Comment:
this fixes a bug that was driving me nuts, and I came across it while I was trying to test this PR. 

if you go into staging/prod, you cannot update a user and it fails silently (try updating the title for this user for example https://moped.austinmobility.io/moped/staff/edit/224). the error says user_id isnt in the mutation, and its because of permissions issues 🙃

Reply:
interesting—why is it necessary to insert the `user_id` if it is generated by DB sequence?

Reply:
good point -- I initially removed it from the mutation but still kept getting the error and then went down the permissions rabbit hole. It is coming from the form submission I think, let me check why its in here at all.

Reply:
@chiaberry I'll wrap this review in the morning, but don't these permissions need to be removed now that the non-login user form payload is fixed?